src/app/globals.css

```
@@ -1382,3 +1382,10 @@ input[type="range"].appearance-none::-moz-range-track {
     scroll-behavior: auto !important;
   }
 }
+
+/* Live-recording dot in the chat composer. A capture must be visible at a
+   glance; a static dot reads as decoration, a pulsing one reads as "on". */
+@keyframes clawPulse {
```

Stylelint (17.14.0)

[error] Expected keyframe name "clawPulse" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)